x-pack/plugins/lens/public/indexpattern_plugin/indexpattern.tsx

```diff
@@ -79,7 +89,19 @@ export const indexPatternDatasource: Datasource<
   },
 
   renderDataPanel(domElement: Element, props: DatasourceDataPanelProps<IndexPatternPrivateState>) {
-    render(<div>Index Pattern Data Source</div>, domElement);
+    render(
+      <div>
+        Index Pattern Data Source
+        <div>
+          {props.state &&
```

Comment:
This check should not be necessary as the frame will make sure initialize finishes before render is called